### PR TITLE
Feat 2.5 fixes

### PR DESCRIPTION
**Pull request by jjos2372**
- More fixes for 2.5 to run on recent Java versions (tested with Java 11).
 - Start logger before the other services to get early exceptions handled and shown.
 - Also min peer version now also using FluxCapacitor.

### CODE REVIEW

File: src/brs/fluxcapacitor/FluxCapacitorImpl.java [hunk 1]
```diff
@@ -6,14 +6,15 @@
...
+import java.util.concurrent.ConcurrentHashMap;
 
 public class FluxCapacitorImpl implements FluxCapacitor {
 
     private final PropertyService propertyService;
     private final Blockchain blockchain;
 
     // Map of Flux Value -> Change Height -> Index of ValueChange in FluxValue. Used as a cache.
-    private final Map<FluxValue<?>, Map<Integer, Integer>> valueChangesPerFluxValue = new HashMap<>();
+    private final Map<FluxValue<?>, Map<Integer, Integer>> valueChangesPerFluxValue = new ConcurrentHashMap<>();
```
Comment (harry1453) [inline src/brs/fluxcapacitor/FluxCapacitorImpl.java:17]: These are the kinda gotchas I mentioned caused by the changes in the runtime

Comment (jjos2372) [reply src/brs/fluxcapacitor/FluxCapacitorImpl.java:17]: I've found this one and that other for DB stuff with the *lambda* expression being rejected on run-time.

File: conf/brs-default.properties [hunk 2]
```diff
@@ -293,6 +293,15 @@ DEV.dumpPeersVersion =
 # Force re-build of derived objects tables at start.
 DEV.forceScan = off
 
+# Current public Testnet settings
+DEV.digitalGoodsStore.startBlock = 0
```
Comment (harry1453) [inline conf/brs-default.properties:297]: Why add this in the default settings? It clutters it up, if we want to change this permanently, this should be removed from default properties and just changed in the Flux Values

Comment (jjos2372) [reply conf/brs-default.properties:297]: Good point, the idea was to make the settings on the *official* testnet the default. So there is no need to check on other places (other than turning the testnet flag on). If it is better to have it hard-coded in the Flux values, also fine by me. Just let me know and I update accordingly.

**Comment by jjos2372** [comment]
I'll close this and adjust to @harry1453 latest PR and submit for review again.

**Pull request closed without merging**